devApps/AdalUAPTestApp/AdalUAPTestApp.csproj

```diff
@@ -17,7 +17,8 @@
     <FileAlignment>512</FileAlignment>
     <ProjectTypeGuids>{A5A43C5B-DE2A-4C0C-9213-0A381AF9435A};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
     <WindowsXamlEnableOverview>true</WindowsXamlEnableOverview>
-    <PackageCertificateKeyFile>UAPTestApp_TemporaryKey.pfx</PackageCertificateKeyFile>
+    <PackageCertificateKeyFile>AdalUAPTestApp_TemporaryKey.pfx</PackageCertificateKeyFile>
```

Comment:
Our certificate expired and I stopped the ADAL builds from working so I had to recreate it. This should be ok right @bgavrilMS?

Reply:
Yes.